### PR TITLE
[lua]Reward proper alignment of if statement

### PR DESCRIPTION
**Pull request by hooksta4**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Adjusted IF statement to align with standards.

## Steps to test these changes
Not applicable.

### CODE REVIEW

File: scripts/actions/abilities/reward.lua [hunk 1]
```diff
@@ -16,7 +16,10 @@ abilityObject.onAbilityCheck = function(player, target, ability)
         return xi.msg.basic.NO_EFFECT_ON_PET, 0
     else
         local id = player:getEquipID(xi.slot.AMMO)
-        if id >= 17016 and id <= 17023 then
+        if
+            id >= 17016 and
+            id <= 17023
+        then
```
Comment (Xaver-DaRed) [inline scripts/actions/abilities/reward.lua:25]: While you are in this very file...

Could you do the same to line 15 and add those item IDs to the enum file?

Comment (hooksta4) [reply scripts/actions/abilities/reward.lua:25]: Sure thing. Done.

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/enum/item.lua [hunk 2]
```diff
@@ -864,6 +864,7 @@ xi.item =
     CATHEDRAL_TAPESTRY                  = 1662,
     CLUSTER_CORE                        = 1667,
     CLEAN_SNAP_ROD                      = 1668,
+    PINCH_OF_HOARY_BOMB_ASH             = 1669,
```
Comment (Xaver-DaRed) [inline scripts/enum/item.lua:867]: This is unused, but oh well...

Comment (hooksta4) [reply scripts/enum/item.lua:867]: Ah yea - i was working on another quest and apparently it slipped in there. It's coming ;)

**Pull request merged**